canary/canary_easy-punctuation_en_us.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard bottom_row="false">

```

Comment:
With bottom row disabled, you won't have the space bar, ctrl, fn, etc..

Reply:
Ah, sorry, my bad. Not sure how that got there. I deleted that now.